frontend/src/metabase/parameters/components/ParameterWidget/ParameterWidget.styled.ts

```diff
@@ -0,0 +1,27 @@
+import styled from "@emotion/styled";
+import FieldSet from "metabase/components/FieldSet";
+import { color } from "metabase/lib/colors";
+
+export const StyledFieldSet = styled(FieldSet)`
```

Comment:
This is the same style previously defined in `frontend/src/metabase/parameters/components/ParameterWidget.css`. Nothing has been changed.